pre_commit/main.py

```diff
@@ -44,8 +44,26 @@ def main(argv=None):
             'in the config file.'
         ),
     )
+    install_parser.add_argument(
+        '-t', '--hook-type', choices=('pre-commit', 'pre-push'),
+        default='pre-commit',
+        help=(
```

Comment:
I think you can remove the help message here, argparse basically duplicates what you've said here